libpod/networking_freebsd.go

```diff
@@ -166,14 +166,23 @@ func (r *Runtime) createNetNS(ctr *Container) (n *jailNetNS, q map[string]types.
 	jconf.Set("allow.raw_sockets", true)
 	jconf.Set("allow.chflags", true)
 	jconf.Set("securelevel", -1)
-	if _, err := jail.Create(jconf); err != nil {
+	j, err := jail.Create(jconf)
+	if err != nil {
 		logrus.Debugf("Failed to create vnet jail %s for container %s", ctrNS.Name, ctr.ID())
 	}
 
 	logrus.Debugf("Created vnet jail %s for container %s", ctrNS.Name, ctr.ID())
 
 	var networkStatus map[string]types.StatusBlock
 	networkStatus, err = r.configureNetNS(ctr, ctrNS)
+	if err != nil {
+		jconf := jail.NewConfig()
+		jconf.Set("persist", false)
+		if err := j.Set(jconf); err != nil {
```

Comment:
Will this crash if jail.Creae(jconf) above threw an error? Won't j be nil?

Reply:
Yes it will. We should return the error if Create fails.